segfast/memmap_loader.py

```diff
@@ -403,7 +418,7 @@ def read_chunk(path, shape, offset, dtype, headers, start, chunk_size):
 
     buffer = np.empty((chunk_size, len(headers)), dtype=np.int32)
```

Comment:
Unfortunately, we can't use `int32` as a dtype of the buffer anymore: now we allow to request for e.g. float headers. Probably the proper way is to create a buffer with `mmap_trace_headers_dtype` without all `unused*` fields and then cast each column to its "native" counterpart upon `DataFrame` creation (it handles conversion automatically). The same applies to a buffer created in `load_headers`.